Makefile

```
@@ -776,8 +776,7 @@ docopen: $(apidocs_html)
 
 .PHONY: docserve
 docserve: $(apidocs_html)
```

```suggestion
docserve: $(apidocs_html)
```

This is not doing what we think it is.

---

```suggestion
docserve: $(apidocs_html) $(apiassets)
```

To have styles available.